dev/tasks/python-wheels/win-build.bat

```diff
@@ -110,6 +110,6 @@ set ARROW_TEST_DATA=%ARROW_SRC%\testing\data
 %PYTHON_INTERPRETER% -c "import pyarrow.dataset" || exit /B
 
 @rem run the python tests, but disable the cython because there is a linking
-@rem issue on python 3.8
+@rem issue on python 3.9
 set PYARROW_TEST_CYTHON=OFF
```

Comment:
I don't know if this still applies, but I figured it probably would

Reply:
Sounds good to me, thanks for kicking things off. I had also create https://github.com/ursa-labs/ursabot/pull/205 to adjust buildbot since I realized 3.9 wasn't being built just by me creating this PR.